src/components/FilterListInput.tsx

```diff
@@ -0,0 +1,31 @@
+import { ChangeEvent } from "react";
+
+interface FilterListProps {
+	searchTerm: string;
+	setSearchTerm: (term: string) => void;
+}
+
+export function FilterListInput({
+	searchTerm,
+	setSearchTerm,
+}: FilterListProps) {
+	const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+		setSearchTerm(e.target.value);
+	};
+
```

Comment:
`nitpick` x 2. Minor change. What do you guys think about adding an onSubmit() to the form to prevent the default refresh in case a user hits enter? I tried clicking enter now after searching for a list item, and the whole page refreshes since its a form element. In the event a user clicks enter, handling the event to remain on the page could be beneficial for a seamless user experience? 
<img width="556" alt="Screenshot 2024-09-01 at 12 59 26 AM" src="https://github.com/user-attachments/assets/c5dfd461-0dcb-4b1f-ba75-e639ad1b7ea8">

Reply:
Great catch!